images/peerjs.svg

```diff
@@ -0,0 +1,25 @@
+<svg width="600" height="300" viewBox="0 0 600 300" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Typically, we don't add the name of the extension to the banner, but it's still fine.